site/_resources/faq.md

```diff
@@ -42,6 +42,18 @@ The best workaround for this Safari issue is to avoid the use of wildcard certif
 If wildcard certificates cannot be avoided, the other workaround is to disable HTTP/2 support which will prevent inappropriate TLS session re-use.
 HTTP/2 support can be disabled by setting the `default-http-versions` field in the Contour [configuration file][3].
 
+## Q: Contour is running but the Envoy container not listening. What is happenning?
```

Comment:
```suggestion
## Q: Contour is running but the Envoy container not listening. What is happening?
```

Reply:
Good catch! I retitled the question to make it clearer.